docker/Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM alpine:3.3
```

Comment:
I believe alpine has issues in kubernetes due to dns resolution (doesn't respect search domains in /etc/resolv.conf). We also probably want the diagnostic tools you get in ubuntu.